testpmd/hooks/roles/apply-cnf-cert-config/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+app_ns: "{{ dci_openshift_app_ns|default('example-cnf') }}"
+operators_regexp: "{{ tnf_operators_regexp|default('cnf-app-mac|testpmd-operator') }}"
+cnfs_regexp: "{{ tnf_cnfs_regexp|default('testpmd-operator|testpmd-app') }}"
+exclude_connectivity_regexp: "{{ tnf_exclude_connectivity_regexp|default('.*') }}"
```

Comment:
I'm unsure about this one, wouldn't this label all the nodes with this default? btw, I'm assuming the defaults are used in here, not sure if any of these setting is being overridden when checking it